components/Chips/examples/ChipsFilterExampleViewController.m

```diff
@@ -12,25 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#import "supplemental/ChipsExamplesSupplemental.h"
-
 #import "MaterialChips+Theming.h"
 #import "MaterialChips.h"
 #import "MaterialContainerScheme.h"
 
 #import "supplemental/ChipsExampleAssets.h"
 
-@implementation ChipsFilterExampleViewController {
-  UICollectionView *_collectionView;
-  NSMutableArray *_selectedIndecies;
-  BOOL _isOutlined;
-}
+@interface ChipsFilterExampleViewController
+    : UIViewController <UICollectionViewDelegate, UICollectionViewDataSource>
+@property(nonatomic, strong) NSArray<NSString *> *titles;
+@property(nonatomic, strong) id<MDCContainerScheming> containerScheme;
+
+@property(nonatomic) UICollectionView *collectionView;
```

Comment:
Nit: Please indicate `strong`/`copy`/`assign` uniformly on the properties for self-consistency.

Reply:
Done.